enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -0,0 +1,202 @@
+""" Models for subsidy_access_policy """
+
+from abc import abstractmethod
+from uuid import uuid4
+
+from django.db import models
+from django_extensions.db.models import TimeStampedModel
+from simple_history.models import HistoricalRecords
+
+from enterprise_access.apps.subsidy_access_policy.constants import AccessMethods
+from enterprise_access.apps.subsidy_access_policy.mocks import catalog_client, group_client, subsidy_client
+
+
+class SubsidyAccessPolicy(TimeStampedModel):
+    """
+    Tie together information used to control access to a subsidy.
+    This abstract model joins group, catalog, and access method.
+
+    .. no_pii:
+    """
+    class Meta:
+        """
+        Metaclass for SubsidyAccessPolicy.
+        """
+        abstract = True
+
+    uuid = models.UUIDField(
+        primary_key=True,
+        default=uuid4,
+        editable=False,
+        unique=True,
+    )
+    version = models.IntegerField(default=0)
```

Comment:
I *think* such a version field would overlap in purpose with django simple history, which I think I'd rather see instead of an explicit version field.